test/linalg/special.jl

```diff
@@ -1,59 +1,54 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 using Base.Test
-debug = false
 
 n= 10 #Size of matrix to test
 srand(1)
 
-debug && println("Test interconversion between special matrix types")
-let a=[1.0:n;]
-   A=Diagonal(a)
-   for newtype in [Diagonal, Bidiagonal, SymTridiagonal, Tridiagonal, Matrix]
-       debug && println("newtype is $(newtype)")
+@testset "Interconversion between special matrix types" begin
+   a = [1.0:n;]
+   A = Diagonal(a)
+   @testset for newtype in [Diagonal, Bidiagonal, SymTridiagonal, Tridiagonal, Matrix]
```

Comment:
looks like you got the later block in this file, but this first interconversion block is also oddly 3-spaced